.github/workflows/test_api_and_frontend.yaml

```diff
@@ -6,6 +6,10 @@ on:
       - master
   pull_request:
 
+strategy:
```

Comment:
I think where you had it before was correct.

Reply:
Moved back to where it was